sdk/deviceupdate/arm-deviceupdate/CHANGELOG.md

```diff
@@ -1,15 +1,17 @@
 # Release History
+    
+## 2.0.0 (2023-01-09)
+    
+**Features**
 
-## 1.0.1 (Unreleased)
+  - Added Type Alias SKU
+  - Added function getContinuationToken
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+**Breaking Changes**
 
+  - Type of parameter sku of interface Account is changed from Sku to SKU
```

Comment:
Even this breaking is introded by fix https://github.com/Azure/autorest.typescript/issues/1404.

But considering there is only one breaking I would prefer to revert this breaking.